### ISSUE TITLE
Image Editor doesn't work with WEBP (when not using imagemagick)

### ISSUE THREAD

**Issue by jahia-ci** [Jahia/jcontent#1963]
> Imported from Jira, on Thu Jan 30, 2025
> Issue: [BACKLOG-22601](https://jira.jahia.org/browse/BACKLOG-22601) in project: DX BACKLOG
> Priority: Major Type: Bug
> Reporter: @romain-pm (Romain Gauthier)
> Assignee: _None found in Jira, making reporter the assignee in GitHub_
> Created: Mon Apr 8, 2024, last updated: Mon Jan 13, 2025
> Status: Open
> **Parent Epic**: [BACKLOG-22734 - Clearer support for different image formats (webp, gif, svg, ..)](https://jira.jahia.org/browse/BACKLOG-22734) (Open)
>
> **Links**:
> - relates to: [SUP-313 - Status of webp support](https://github.com/Jahia/jira-archives/issues/49260) *[[JIRA](https://jira.jahia.org/browse/SUP-313)]* (Closed)
>
>
> **Attachments**:
> - [image-2024-04-08-19-25-09-985.png](https://jira-artifacts.jahia.com/jira/BACKLOG/BACKLOG-22601/image-2024-04-08-19-25-09-985.png) _uploaded by: @romain-pm (Romain Gauthier) (303.35 KB - Apr 8, 2024_)
> - [image-2024-04-08-19-26-09-732.png](https://jira-artifacts.jahia.com/jira/BACKLOG/BACKLOG-22601/image-2024-04-08-19-26-09-732.png) _uploaded by: @romain-pm (Romain Gauthier) (288.5 KB - Apr 8, 2024_)
> - [image-2024-04-08-19-26-56-569.png](https://jira-artifacts.jahia.com/jira/BACKLOG/BACKLOG-22601/image-2024-04-08-19-26-56-569.png) _uploaded by: @romain-pm (Romain Gauthier) (330.19 KB - Apr 8, 2024_)
> - [image-2024-04-08-19-27-34-318.png](https://jira-artifacts.jahia.com/jira/BACKLOG/BACKLOG-22601/image-2024-04-08-19-27-34-318.png) _uploaded by: @romain-pm (Romain Gauthier) (408.19 KB - Apr 8, 2024_)
>

### Context

Webp images display well in jcontent, pickers and are now part of the demo website so we should fully support them. 

Note: BUG only reproducible without imagemagick
### Steps to reproduce

upload a webp image to jcontent 

open it in image editor

try any of crop, resize, rotate

 
### Current Behavior

different flavors of error 

[image-2024-04-08-19-25-09-985\.png](https://jira-artifacts.jahia.com/jira/BACKLOG/BACKLOG-22601/image-2024-04-08-19-25-09-985.png)

 

[image-2024-04-08-19-26-09-732\.png](https://jira-artifacts.jahia.com/jira/BACKLOG/BACKLOG-22601/image-2024-04-08-19-26-09-732.png)
### Desired Behavior

it should work

**Comment by Fgerthoffert** [comment]
I thought ImageMagick was not included in our docker images ([code](https://github.com/Jahia/jahia-private/blob/d67f9caa35420005e33949adb36da5b6b3d1160b/docker/docker-jahia-core/Dockerfile#L10)).

**Comment by romain-pm** [comment]
you're right, ticket to follow that up  https://github.com/Jahia/jahia-private/issues/4159

**Comment by romain-pm** [comment]
Closing as won't do since we consider that imagemagick is now required in production
